.travis.yml

```diff
@@ -19,11 +19,9 @@ matrix:
   - go: 1.8.2
     env: COVERAGE="true"
 script:
-- go fmt $(go list ./... | grep -v vendor) | wc -l | grep 0
-- go vet $(go list ./... | grep -v vendor)
```

Comment:
I think the `go fmt` is not yet put into a script, right?
It's in a PR #896 which would need update and rebase.

Reply:
Yup. That's what I was trying to get at with my final bullet in my description above.

Reply:
Yeah, but instead of removing the go fmt line in this PR/commit, we should leave it in the travis file and remove it when the go fmt script is added, right?

Reply:
Sure. Will update.